src/components/modal.jsx

```
@@ -52,66 +52,66 @@
-                <div className='affection--bar-container'>
-                  <div className='affection--bar' style={{ '--affectionWidth': `${energyLevel}%` }}>.</div>
+              <div className='mt-4'>
+                <p className='font-light text-sm'>Energy Friendly</p>
+                <div className='w-full bg-yellow-200 rounded h-2.5 mt-1'>
+                  <div className='bg-yellow-500 h-2.5 rounded' style={{ width: `${energyLevel}%` }}></div>
                 </div>
               </div>
             
             </div>
-            <div className='learn-more'><FaInfoCircle className='tip--icon'/>Read more about {breedName} <a href={wikipediaURL} target='_blank'>here</a>!</div>
+            <div className='mt-4'><FaInfoCircle className='inline-block mr-1'/>Read more about {breedName} <a href={wikipediaURL} target='_blank' className='text-blue-500 hover:underline'>here</a>!</div>
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)